api/src/wfl/module/sg.clj

```diff
@@ -191,21 +191,21 @@
 
 ;; visible for testing
 (defn register-workload-in-clio
-  "Use `tx` to register `_workload` outputs with Clio."
-  [{:keys [executor items output] :as _workload} tx]
-  (run!
-   (partial register-workflow-in-clio executor output)
-   (postgres/get-table tx items)))
+  [{:keys [executor output workflows] :as _workload}]
+  (run! (partial register-workflow-in-clio executor output) workflows))
```

Comment:
Restore the docstring or remove `:as _workload`.

Reply:
I don't understand when you want a `_workload`? I thought it was for more than just the docstring.

Reply:
I don't always understand either.  Sometimes I include it to clarify what the keywords belong to.  Sometimes it's to support the docstring.  Sometimes it's both.

Reply:
BTW: Thank you for restoring the docstring.